web/packages/teleport/src/Apps/MCPAppConnectDialog.tsx

```diff
@@ -78,14 +95,41 @@ export function MCPAppConnectDialog(props: { app: App; onClose: () => void }) {
               <Text bold as="span">
                 Step 2
               </Text>
-              {' - Log in the MCP server'}
+              {' - Configure your MCP client'}
             </Text>
-            <TextSelectCopy text={`tsh mcp login ${app.name}`} />
+            <Flex alignItems="center" justifyContent="left" columnGap={2}>
+              <Link href={links.cursor} target="_blank">
+                <ResourceIcon name="mcpCursor" height="32px" />
+              </Link>
+              <Link href={links.vscode} target="_blank">
+                <ResourceIcon name="mcpVscode" height="25px" />
+              </Link>
+              <Link href={links.vscodeInsiders} target="_blank">
+                <ResourceIcon name="mcpVscodeInsiders" height="25px" />
+              </Link>
+            </Flex>
+            <Box>
+              Here is a sample Claude Desktop config to connect to this MCP
+              server:
+            </Box>
+            <TextSelectCopyMulti
+              bash={false}
+              lines={[
+                {
+                  text: claudeConfig,
+                },
+              ]}
+            />
+            <Box>
+              Alternatively, run the following to generate the config from the
+              command line.
+            </Box>
+            <TextSelectCopy text={`tsh mcp config ${app.name}`} />
+            <Box>
+              *Note: You might need to restart your MCP client to load the
+              updated configuration.
```

Comment:
As someone who has no idea what `tsh mcp config` does, I don't know what "the updated configuration" means in this context. 😅 Does it mean that `tsh mcp config` has some side effects and that's why I need to restart the MCP client?

On the other hand, just above this note there's a config that I can copy, so it's just a bit confusing to me why I need to restart anything in the first place since I can just copy something? 🤔

I guess I just don't fully understand the relationship between the buttons, the config and the tsh command.